components/camel-as2/camel-as2-api/src/main/java/org/apache/camel/component/as2/api/protocol/ResponseMDN.java

```diff
@@ -16,40 +16,13 @@
  */
 package org.apache.camel.component.as2.api.protocol;
 
-import java.io.IOException;
-import java.io.StringWriter;
-import java.security.PrivateKey;
-import java.security.cert.Certificate;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Properties;
-
-import org.apache.camel.component.as2.api.AS2AsynchronousMDNManager;
-import org.apache.camel.component.as2.api.AS2Charset;
-import org.apache.camel.component.as2.api.AS2Constants;
-import org.apache.camel.component.as2.api.AS2Header;
-import org.apache.camel.component.as2.api.AS2ServerManager;
-import org.apache.camel.component.as2.api.AS2SignatureAlgorithm;
-import org.apache.camel.component.as2.api.AS2SignedDataGenerator;
-import org.apache.camel.component.as2.api.AS2TransferEncoding;
-import org.apache.camel.component.as2.api.InvalidAS2NameException;
-import org.apache.camel.component.as2.api.entity.AS2DispositionModifier;
-import org.apache.camel.component.as2.api.entity.AS2DispositionType;
-import org.apache.camel.component.as2.api.entity.DispositionMode;
-import org.apache.camel.component.as2.api.entity.DispositionNotificationMultipartReportEntity;
-import org.apache.camel.component.as2.api.entity.DispositionNotificationOptions;
-import org.apache.camel.component.as2.api.entity.DispositionNotificationOptionsParser;
-import org.apache.camel.component.as2.api.entity.MultipartSignedEntity;
+import org.apache.camel.component.as2.api.*;
+import org.apache.camel.component.as2.api.entity.*;
```

Comment:
Please avoid the * imports. This will report warning with the sourcecheck profile enabled.